packages/common/src/ui/forms/Modal/ModalErrorMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
This is really similar to just using the `helperText` prop with a few style changes. Maybe we just use that in `ModalInput`?

https://codesandbox.io/s/validationtextfields-material-demo-forked-inzd9?file=/demo.js